src/frontend/posts_page/components/PostsContainer.tsx

```diff
@@ -1,12 +1,12 @@
 import React from "react";
-import PostBlock, {Post} from './PostBlock';
+import PostBlock, {IPost} from './PostBlock';
 
 
-interface PostsLayoutProps {
-  posts: Post[];
+interface IPostsContainerProps {
+  posts: IPost[];
 }
 
-class PostsLayout extends React.Component<PostsLayoutProps, {}> {
+class PostsContainer extends React.Component<IPostsContainerProps, {}> {
```

Comment:
Reason for renaming: I think this class feels more like a container that holds all the posts. Layout sounds more like styling for all posts.